src/bin/vip-wp.js

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+// @flow
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+import { stdout } from 'single-line-log';
+import SocketIO from 'socket.io-client';
+import IOStream from 'socket.io-stream';
+
+/**
+ * Internal dependencies
+ */
+import API, { API_HOST } from 'lib/api';
+import app from 'lib/api/app';
+import command from 'lib/cli/command';
+import { formatEnvironment } from 'lib/cli/format';
+import { trackEvent } from 'lib/tracker';
+import Token from '../lib/token';
+
+const appQuery = `id, name, environments {
+	id
+}`;
+
+command( {
+	requiredArgs: 2,
+	appContext: true,
+	childEnvContext: true,
```

Comment:
@elazzabi we need to change this back to `envContext` - `childEnvContext` disallows running on prod, which is not what we want here.

Reply:
yes, just reverted it 🙌